content/attacks/posts/2023-05-01-Level-Finance.md

```diff
@@ -13,7 +13,7 @@ On May 1, 2023, Level Finance, a decentralized finance (DeFi) protocol, was hack
 
 The identity of the attacker is unknown.
 
-brc-20:
+bsc-20:
```

Comment:
Leave as 'BSC' please. [FYI](https://www.google.com/search?q=what+is+bsc+and+bep20+difference)